braindecode/regressor.py

```diff
@@ -185,3 +185,42 @@ def _default_callbacks(self):
             ),
             ("print_log", PrintLog()),
         ]
+
+    def predict_proba(self, X):
```

Comment:
to me it does not make sense to have a predict_proba for a regressor

Reply:
`predict_proba` is already implemented in skorch, so this changes the behavior in cropped mode to return averaged predictions per trial not raw predictions per crop. After the change it would be consistent with `EEGClassifier` `predict_proba` and `predict` behavior.